docs/spec/spec.md

```diff
@@ -9,10 +9,93 @@ These are Kubernetes resources that been introduced using Custom Resource
+| conditions         | Conditions  | Broker conditions.                                                                           |             |
+
+##### Conditions
+
+- **Ready.** True when the Broker is provisioned and ready to accept events.
+- **Addressable.** True when the Broker has an resolved address in it's status.
+
+#### Events
+
+- BrokerReconciled
+- BrokerUpdateStatusFailed
+- IngressSubscriptionDeleteFailed
+- IngressSubscriptionCreateFailed
+
+---
+
+## kind: Trigger
+
+### group: eventing.knative.dev/v1alpha1
+
+_A Trigger represents subscriber of events with a filter for a specific broker._
+
+### Object Schema
+
+#### Spec
+
+| Field        | Type           | Description                                                                                                                                                                 | Constraints |
+| ------------ | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| broker       | String         | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                         |             |
+| filter       | TriggerFilter  | Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscripting to all events. |             |
+| subscriber\* | SubscriberSpec | Subscriber is the addressable that receives events from the Broker that pass the Filter.                                                                                    |             |
+
+\*: Required
+
+#### Status
+
+| Field              | Type        | Description                                                                                              | Constraints |
+| ------------------ | ----------- | -------------------------------------------------------------------------------------------------------- | ----------- |
+| observedGeneration | int64       | The 'Generation' of the Broker that was last processed by the controller.                                |             |
+| subscriberURI      | Addressable | Address of the subscribing endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
+| conditions         | Conditions  | Trigger conditions.                                                                                      |             |
+
+##### Conditions
+
+- **Ready.** True when the Trigger is provisioned and configuration is ready to
+  deliver events to the subscriber.
+- **BrokerExists.** True when the Broker can be retrieved.
+- **Subscribed.** True when the subscriber is subscribed to the Broker.
+
+#### Events
+
+- TriggerReconciled
+- TriggerReconcileFailed
+- TriggerUpdateStatusFailed
+- SubscriptionDeleteFailed
```

Comment:
I would probably omit both the `Subscription` events. I think they are implementation details that don't need to be elevated to the spec.